react/README.md

```diff
@@ -4,16 +4,312 @@
 
 ## Table of Contents
 
-  1. [Basic Rules](#basic-rules) #TBD
-  1. [Naming](#naming) #TBD
-  1. [Declaration](#declaration) #TBD
-  1. [Alignment](#alignment) #TBD
-  1. [Quotes](#quotes) #TBD
-  1. [Spacing](#spacing) #TBD
-  1. [Props](#props) #TBD
-  1. [Parentheses](#parentheses) #TBD
-  1. [Tags](#tags) #TBD
-  1. [Methods](#methods) #TBD
-  1. [Ordering](#ordering) #TBD
+  1. [Basic Rules](#basic-rules)
+  1. [Naming](#naming)
+  1. [Declaration](#declaration)
+  1. [Alignment](#alignment)
+  1. [Quotes](#quotes)
+  1. [Spacing](#spacing)
+  1. [Props](#props)
+  1. [Parentheses](#parentheses)
+  1. [Tags](#tags)
+  1. [Methods](#methods)
+  1. [Ordering](#ordering)
+
+## Basic Rules
+
+  - Only include one React component per file.
```

Comment:
I'd change this to exposing one React component per file through a default export. This allows for smaller, private components to be defined for usage in the actual component that's exported.